src/lib/middleware/sharifyLocals.ts

```diff
@@ -28,5 +28,22 @@ export function sharifyLocalsMiddleware(
       : undefined
   res.locals.sd.ARTSY_XAPP_TOKEN = artsyXapp.token
 
+  // Determines device type for non-responsive pages.
+  res.locals.sd.IS_MOBILE = Boolean(
+    (ua.match(/iPhone/i) && !ua.match(/iPad/i)) ||
+      (ua.match(/Android/i) && ua.match(/Mobile/i)) ||
+      ua.match(/Windows Phone/i) ||
+      ua.match(/BB10/i) ||
+      ua.match(/BlackBerry/i)
+  )
+
+  res.locals.sd.IS_TABLET = Boolean(
+    (ua.match(/iPad/i) && ua.match(/Mobile/i)) ||
+      // specifically targets Vivo
+      (ua.match(/vivo/i) && ua.match(/Mobile/i)) ||
+      // targets android devices that are not mobile
+      (ua.match(/Android/i) && ua.match(/Mobile/i))
+  )
```

Comment:
Strictly speaking, I didn't _need_ to move this here. But it made sense to me that these 2 settings would stay close together. It's only use appears to be here:

https://github.com/artsy/force/blob/953d162ee90f7a2223d34ddf902a1c888f0bd2f8/src/desktop/apps/article/routes.tsx#L159-L160